package.json

```diff
@@ -20,7 +20,7 @@
     "lint:style": "stylelint **/*.{vue,css}",
     "lint": "yarn lint:js && yarn lint:style && yarn lint:markdown",
     "test": "./node_modules/.bin/cypress open",
-    "test:unit": "jest",
+    "test:unit": "NODE_OPTIONS=--experimental-vm-modules jest",
```

Comment:
`rehype-stringify` is now an ESM-only package. We need to invoke jest with experimental module support.